oaknational/Oak-Web-Application
# PUPIL-756/fix(units): fix unit text for legacy and cycle 1 unit list

## Description

Music year: 1965

- List of changes

## Expected behaviour

- **New** curriculum unit title should read “**Choose a unit**“
- **Legacy** curriculum unit title should read “**Choose a previously released unit**“

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2591--oak-web-application.netlify.thenational.academy/pupils/programmes/art-primary-year-1-l/units
1. Go to https://deploy-preview-2591--oak-web-application.netlify.thenational.academy/pupils/programmes/geography-primary-year-4/units
1. Go to https://deploy-preview-2591--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-secondary-year-10-foundation-l/units
4. You should see the counter text is either previously released.. or just choose a unit depending on whether your on a legay or cycle 1 programme page

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
